src/ffi/defaults_builder.rs

```diff
@@ -33,6 +33,7 @@ impl FeaturesDefaultsBuilder {
             multicast: false,
             ipv6: false,
             nicknames: false,
+            batching: default(),
```

Comment:
NIT: Please explicitly use None, so it would be clear it's not some unknown defaults

Reply:
Good point. I like that too. Updated :)